internal/integration_test/vs/bench_allocation.go

```diff
@@ -50,14 +50,25 @@ func allocationCall(m Module, _ int) error {
 		return err
 	}
 
-	// Now, we can call "greet", which reads the string we wrote to memory!
-	if err = m.CallI32I32_V(testCtx, "greet", namePtr, nameSize); err != nil {
-		return err
+	// Now, we can call "greeting", which reads the string we wrote to memory!
+	ptrSize, fnErr := m.CallI32I32_I64(testCtx, "greeting", namePtr, nameSize)
```

Comment:
so the main thing here is that the benchmark is doing something different than before. Due to this, it cannot be compared directly to the prior (now it is calling "greeting" but before it was calling "greet"